scripts/run-nat-node.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+function echoerr() { echo "$@" 1>&2; }
+
+if [ $(id -u) -ne 0 ]; then 
+  echoerr "ERR: Please run as root!"
+  exit 1 
+fi
+
+
+declare rundir="/var/hoprd"
+unset RELEASE API_TOKEN
+
+function usage {
+  cat <<EOF
+Usage: $(basename "$0") [OPTION]
+
+  -r VALUE    relase name, e.g. prague
+  -t VALUE    API token
+  -h          display help
+EOF
+
+  exit 2
+}
+
+
+while getopts "r:t:h" optKey; do
+  case "$optKey" in
+    r)
+      RELEASE=$OPTARG
+      ;;
+    t)
+      API_TOKEN=$OPTARG
+      ;;
+    h|*)
+      usage
+      ;;
+  esac
+done
+
+shift $((OPTIND - 1))
+
+[ -z "$RELEASE" ] && usage
+[ -z "$API_TOKEN" ] && usage
+
+
+cat <<EOF >docker-compose.yaml
+version: "3.9"
+
+# Start an internal-only bridge network to simulate NAT (NATwork)
+networks:
+  natwork:
+    driver: bridge
+
+# Starts HOPRD behind NAT
+services:
+  hoprd-natted:
+    
+    image: gcr.io/hoprassociation/hoprd:$RELEASE
+
+    command: 
+      - "--admin"
+      - "--adminHost"
+      - "0.0.0.0"
+      - "--announce"
+      - "--healthCheck"
+      - "--healthCheckHost"
+      - "0.0.0.0"
+      - "--identity"
+      - "/app/db/.hopr-identity"
+      - "--init"
+      - "--rest"
+      - "--restHost"
+      - "0.0.0.0"
+      - "--environment"
+      - "$RELEASE"
+      - "--apiToken"
+      - "$API_TOKEN"
+      - "--password"
+      - "pw14775124087585pw"
+   
+    volumes:
+      - "$rundir:/app/db"
+
+    environment:
+      - "DEBUG=hopr*"
+
+    networks:
+      - "natwork"
+
+    ports:
+      - "3000:3000"
+      - "3001:3001"
+EOF
+
+
+# Start Docker compose
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$PWD:$PWD" -w="$PWD" docker/compose:1.29.2 up --force-recreate --abort-on-container-exit --exit-code-from hoprd-natted
+ec=$?
+
+echo "Node exitted with code $ec"
```

Comment:
This is misleading, its not the hoprd node but the compose container which tried to run the node. I'd rather use `docker-compose start` or `docker-compose up -d` since I don't want to keep the compose container running.